### PR TITLE
Parameterizing the `compare_with_acts_seeding` test

### PR DESCRIPTION
**Pull request by beomki-yeo**
This PR parameterizes the `compare_with_acts_seeding` test so it can iterate over all event files in `tml_hits/`.
I also tightened the pass conditions with 100% matching ratio, but I personally don't think that traccc and acts seeding should always satisfy 100% mathing ratio. 
Thus people can mitigate the conditions whenever they need in the future

### CODE REVIEW

**Review by krasznaa — CHANGES_REQUESTED** [review]
I very much like the update. Only have technical requests. :smile:

File: tests/cpu/compare_with_acts_seeding.cpp [hunk 1]
```diff
@@ -387,4 +394,19 @@ TEST(algorithms, compare_with_acts_seeding) {
     std::cout << "number of traccc params: " << traccc_params.size()
               << std::endl;
     std::cout << "params matching ratio: " << params_match_ratio << std::endl;
+    */
```
[the remark this answers is not in the document]
Comment (beomki-yeo) [reply tests/cpu/compare_with_acts_seeding.cpp]: OK

File: tests/cpu/compare_with_acts_seeding.cpp [hunk 2]
```diff
@@ -374,8 +379,10 @@ TEST(algorithms, compare_with_acts_seeding) {
     }
 
     float params_match_ratio = float(n_params_match) / traccc_params.size();
-    EXPECT_TRUE((params_match_ratio > 0.95) && (params_match_ratio <= 1.));
+    EXPECT_EQ(acts_params.size(), traccc_params.size());
+    EXPECT_EQ(params_match_ratio, 1);
```
[the remark this answers is not in the document]
Comment (beomki-yeo) [reply tests/cpu/compare_with_acts_seeding.cpp]: OK

File: tests/cpu/compare_with_acts_seeding.cpp [hunk 3]
```diff
@@ -263,8 +267,9 @@ TEST(algorithms, compare_with_acts_seeding) {
 
     float seed_match_ratio = float(n_seed_match) / seeds.total_size();
 
-    // Ensure that the difference between ACTS and traccc is small enough
-    EXPECT_TRUE((seed_match_ratio > 0.95) && (seed_match_ratio <= 1.));
+    // Ensure that ACTS and traccc give the same result
+    EXPECT_EQ(seeds.get_headers()[0], seedVector.size());
+    EXPECT_EQ(seed_match_ratio, 1);
```
[the remark this answers is not in the document]
Comment (beomki-yeo) [reply tests/cpu/compare_with_acts_seeding.cpp]: RIght. I changed now

**Review by krasznaa — APPROVED** [review]
Looks good!

**Pull request merged**